money_bot/handlers/game_handlers.py

```diff
@@ -0,0 +1,227 @@
+import random
+
+from aiogram import Bot, Dispatcher, executor, types
+from aiogram.utils.callback_data import CallbackData
+
+from money_bot.utils import db_utils
+from money_bot.utils.states import GlobalStates
+
+change_sum = CallbackData("sum", "action", "value")
+b = 0
+a = 0
+
+
+def main_game_keyboard():
+    return types.InlineKeyboardMarkup(row_width=4).add(
+        types.InlineKeyboardButton("1", callback_data=change_sum.new(action="make", value="1")),
+        types.InlineKeyboardButton("10", callback_data=change_sum.new(action="make", value="10")),
+        types.InlineKeyboardButton("50", callback_data=change_sum.new(action="make", value="50")),
+        types.InlineKeyboardButton("100", callback_data=change_sum.new(action="make", value="100")),
+        types.InlineKeyboardButton("1000", callback_data=change_sum.new(action="make", value="1000")),
+        types.InlineKeyboardButton("+ 1", callback_data=change_sum.new(action="plus", value="1")),
+        types.InlineKeyboardButton("+ 5", callback_data=change_sum.new(action="plus", value="5")),
+        types.InlineKeyboardButton("- 1", callback_data=change_sum.new(action="minus", value="1")),
+        types.InlineKeyboardButton("- 5", callback_data=change_sum.new(action="minus", value="5")),
+        types.InlineKeyboardButton(
+            "\U0001F3B2 play \U0001F3B2", callback_data=change_sum.new(action="play", value="-")
+        ),
+        types.InlineKeyboardButton(
+            "\U0001F4DC rules \U0001F4DC", callback_data=change_sum.new(action="show_rules", value="-")
+        ),
+    )
+
+
+def get_menu_button():
+    return types.InlineKeyboardMarkup().row(
+        types.InlineKeyboardButton(
+            "\U0001F519 menu \U0001F519", callback_data=change_sum.new(action="menu", value="-")
+        )
+    )
+
+
+def bitcoin():
+    return types.InlineKeyboardMarkup().row(
+        types.InlineKeyboardButton("\U00002B06 grow \U00002B06", callback_data=change_sum.new(action="up", value="-")),
+        types.InlineKeyboardButton(
+            "\U00002B07 fall \U00002B07", callback_data=change_sum.new(action="down", value="-")
+        ),
+    )
+
+
+def play_again():
+    return types.InlineKeyboardMarkup().row(
+        types.InlineKeyboardButton(
+            "\U0001F3B0 play again! \U0001F3B0", callback_data=change_sum.new(action="play_again", value="-")
+        )
+    )
+
+
+async def entry_point(message: types.Message):
+    bot = Bot.get_current()
+    global a
+    a = (await db_utils.get_user_by_id(message.from_user.id)).money
+    await bot.send_message(
+        message.chat.id,
+        f"\U0001F4B8       You have {a} money now!         \U0001F4B8",
+        reply_markup=main_game_keyboard(),
+    )
+
+
+async def callback_results(query: types.CallbackQuery, callback_data: dict):
+    bot = Bot.get_current()
+    global b, a
```

Comment:
I can't see where you save the `b` i.e. current user's bet in the database. You should save it in mongo, and remove the globals